backend/open_webui/models/users.py

```
@@ -361,16 +562,35 @@ def update_user_last_active_by_id(self, id: str) -> Optional[UserModel]:
+
+                # Update or insert provider entry
+                oauth[provider] = {"sub": sub}
+
+                # Persist updated JSON
+                db.query(User).filter_by(id=id).update({"oauth": oauth})
                 db.commit()
 
-                user = db.query(User).filter_by(id=id).first()
                 return UserModel.model_validate(user)
+
         except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)